spec/models/freak_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Freak, type: :model do
+  context 'with validation test' do
+    let(:user) { build(:freak) } # user.new({})
+
+    it 'is ensures name present' do
+      build(:freak)
```

Comment:
The test does nothing. A `Freak` object is built and that's it. Please read https://www.justinweiss.com/articles/writing-better-tests-with-the-three-phase-pattern/ and apply it here.